karma.conf.js

```diff
@@ -0,0 +1,83 @@
+// Karma configuration
+// Generated on Mon May 15 2017 15:46:55 GMT+0200 (CEST)
+
+module.exports = function ( config ) {
+	config.set( {
+
+		// base path that will be used to resolve all patterns (eg. files, exclude)
+		basePath: '',
+
+		// frameworks to use
+		// available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+		frameworks: [ 'qunit' ],
+
+		// list of files / patterns to load in the browser
+		files: [
+			'node_modules/jquery/dist/jquery.js',
+			'vendor/data-values/javascript/src/dataValues.js',
+			'vendor/data-values/javascript/lib/util/util.inherit.js',
+			'vendor/data-values/javascript/src/DataValue.js',
+			'src/__namespace.js',
+			'src/Claim.js',
+			'src/Entity.js',
+			'src/Fingerprint.js',
+			'src/GroupableCollection.js',
+			'src/Group.js',
+			'src/Map.js',
+			'src/MultiTerm.js',
+			'src/Reference.js',
+			'src/SiteLink.js',
+			'src/Snak.js',
+			'src/Statement.js',
+			'src/Term.js',
+			'src/Set.js',
+			'src/List.js',
+			'vendor/data-values/javascript/lib/globeCoordinate/globeCoordinate.js',
+			'vendor/data-values/javascript/lib/globeCoordinate/globeCoordinate.GlobeCoordinate.js',
+			'vendor/data-values/javascript/src/valueFormatters/valueFormatters.js',
+			'vendor/data-values/javascript/src/valueParsers/valueParsers.js',
+			'vendor/data-values/javascript/src/dataValues.js',
+			'vendor/data-values/javascript/src/*.js',
+			'vendor/data-values/javascript/src/values/*.js',
+			'src/*.js',
```

Comment:
:smile: Seems reasonable to expect that it is generated automatically, but no.
I did it by hand.
Basically, I was adding needed files before `'src/*.js'` until there were no errors. May be we can generate it, who knows...